vendor/modules.txt

```diff
@@ -518,8 +518,7 @@ github.com/moby/ipvs
 # github.com/moby/spdystream v0.2.0 => github.com/moby/spdystream v0.2.0
 github.com/moby/spdystream
 github.com/moby/spdystream/spdy
-# github.com/moby/sys/mountinfo v0.6.0 => github.com/moby/sys/mountinfo v0.6.0
-## explicit
+# github.com/moby/sys/mountinfo v0.6.1 => github.com/moby/sys/mountinfo v0.6.1
```

Comment:
This part also looks wrong, the `explicit` line should not be removed.

Reply:
[e65ce59d421ca705c853619153a8cbc4a880c1d7](https://github.com/kubernetes/kubernetes/pull/109217/commits/57cb3264320f3616e319e4b6af5e142a46913632)

https://github.com/kubernetes/kubernetes/pull/109217/commits/13a66757f6e9ae6d45fd34762ad973be858297b3

These two commits should be squashed as with introduction of this change, its not an indirect dependency anymore.